### DISCUSSION TITLE
Skeleton - Avrundede hjørner

### DISCUSSION THREAD

**Discussion by Murstam** [fremtind/jokul#3520] [Annet]
Jeg syns skeletonloader får et finere og mer behagelig uttrykk med avrundede hjørner og har lyst til å oppdatere det i Jøkul. Noen som er uenig?
<img width="423" alt="Screenshot 2023-05-05 at 15 54 23" src="https://user-images.githubusercontent.com/47025023/236477983-80b6cdd9-cc48-42ef-b08b-46d386ca62b7.png">
<img width="925" alt="Screenshot 2023-05-05 at 15 54 51" src="https://user-images.githubusercontent.com/47025023/236477992-a3305b47-10de-49c6-8f49-baaa0843d8c3.png">

**Comment by ivnai** [comment]
Sikkert bedre @Murstam! 4px round?

**Reply by Murstam** [reply]
Jeg har brukt 2px på det meste, men beholdt 4px på navcard fordi original form har det. Mulig det kunne bare vært 2px/4px på alt? @ivnai

**Reply by ivnai** [reply]
Tenkte akkurat det; 4 på alt, for konsistens sin sake og unngå ekstra informasjon.

**Reply by Slaine2** [reply]
Finner også at den ser mye bedre avrundet ut.

Kort er definitivt bra med 4px og 2px passer for tekstblokkene – så enig med alt 😂

**Reply by ivnai** [reply]
Ikke enig @Slaine2. Hva med andre slags innhold som faller ikke inni de 2 kategori, skal vi utvikle systemet av skeleton roundness? Og hva er poenget med å ha 2 forskjellige, skjønner bruker mening av boks med 2 vs 4?

**Reply by Slaine2** [reply]
Jeg tror i forbindelse med at skeletonloader også kan ligge i en eksisterende kort passer det med 2 nivå bedre. 
Den er min tanken

**Reply by wkillerud** [reply]
Design by committee har valgt å holde oss til én radiusverdi, og det 2px.

Hovedgrunn er at utviklere som oftest bygger opp egen layout med et basis skeleton-element, og vil unngå å måtte ta stilling til radiusverdi hele tiden. Og det er begrenset verdi i å ha flere. De skal forhåpentligvis ikke vises lenge uansett.

https://github.com/fremtind/jokul/pull/3533
